systems/subscriber/subscriber-registry/pkg/server/event.go

```diff
@@ -0,0 +1,49 @@
+package server
+
+import (
+	"context"
+
+	log "github.com/sirupsen/logrus"
+	epb "github.com/ukama/ukama/systems/common/pb/gen/events"
+	pb "github.com/ukama/ukama/systems/subscriber/subscriber-registry/pb/gen"
+	"github.com/ukama/ukama/systems/subscriber/subscriber-registry/pkg/db"
+	"google.golang.org/protobuf/proto"
+	"google.golang.org/protobuf/types/known/anypb"
+)
+
+type SubcriberRegistryEventServer struct {
+	subscriberRepo db.SubscriberRepo
+	epb.UnimplementedEventNotificationServiceServer
+}
+
+func NewSubscriberEventServer(subscriberRepo db.SubscriberRepo) *SubcriberRegistryEventServer {
+	return &SubcriberRegistryEventServer{
+		subscriberRepo: subscriberRepo,
+	}
+}
+
+func (l *SubcriberRegistryEventServer) EventNotification(ctx context.Context, e *epb.Event) (*epb.EventResponse, error) {
+	log.Infof("Received a message with Routing key %s and Message %+v", e.RoutingKey, e.Msg)
+	switch e.RoutingKey {
+	case "event.cloud.simManager.sim.allocation":
```

Comment:
make sure the listened route matches the publishers'

Reply:
I need to  return req for `Sim allocation ` method here but I don't want to merge simManager on my branch now.I want this branch to be merged to fix those small issues.